src/models/fhir/index.ts

```diff
@@ -0,0 +1,3 @@
+import { parse } from "./parse";
```

Comment:
export *

Reply:
Should we still use `export * from "./somewhere"` if we have multiple imports/exports:

```javascript
import { parse } from "./parse";
import { hasRequiredFields } from "./required";

export default { parse, hasRequiredFields };
```

Reply:
yes, also I dont understand why you use export default ?

Reply:
Oh I use export default so that the helper function can be imported and used this way:

```javascript
import fhirHelper from "src/models/fhir"

parsed = fhirHelper.parse(...);
fhirHelper.hasRequiredFields(..., parsed);
```